charts/pokt-portal-v2/values.yaml

```diff
@@ -198,13 +194,23 @@ affinity: {}
 # Environment variable listing
 env:
   # non sensitive variables
-  normal:
-    # remote cache
-    FOO: "bar"
+  normal: []
```

Comment:
You are right about that. The env variable you just mentioned is in the deployment.yaml https://github.com/pokt-foundation/charts/pull/106/files#diff-9f1ccbedb274b0b477b631d8ba18564e252f7d0b34fad9c7aeca0015ffc7af4fR87
But you can modify it as you see fit.

Reply:
Gotcha, well at least I think PORT needs to be renamed to ENV_PORT so the app picks it up.

Reply:
I don't think so. I think the env variables have ENV_ prefix which isn't included in the secret key. @adshmh can you confirm if I'm right?
Also, this worked when I tested locally with the key as PORT rather than ENV_PORT.